airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSourceOperations.java

```diff
@@ -5,16 +5,82 @@
 package io.airbyte.integrations.source.mssql;
 
 import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.microsoft.sqlserver.jdbc.Geography;
+import com.microsoft.sqlserver.jdbc.Geometry;
+import com.microsoft.sqlserver.jdbc.SQLServerResultSetMetaData;
 import io.airbyte.db.jdbc.JdbcSourceOperations;
+import java.sql.JDBCType;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class MssqlSourceOperations extends JdbcSourceOperations {
 
-  protected void putBinary(final ObjectNode node, final String columnName, final ResultSet resultSet, final int index) throws SQLException {
+  private static final Logger LOGGER = LoggerFactory.getLogger(MssqlSourceOperations.class);
+
+  @Override
+  public void setJsonField(final ResultSet resultSet, final int colIndex, final ObjectNode json)
```

Comment:
Please, add some java doc. To clarify why we need to override `setJsonField(final ResultSet resultSet, final int colIndex, final ObjectNode json)` for MSSQL Source

Reply:
done, thanks!